extensions/positron-assistant/src/models.ts

```diff
@@ -1676,9 +1688,10 @@ export async function createAutomaticModelConfigs(): Promise<ModelConfig[]> {
 						name: model.source.provider.displayName,
 						model: model.source.defaults.model,
 						apiKey: result.token,
+						toolCalls: model.source.defaults.toolCalls,
+						completions: model.source.defaults.completions,
```

Comment:
this is the crucial fix! I explicitly set toolCalls/completions for all the providers to ensure we have defaults to apply here